opencti-platform/opencti-front/src/private/components/settings/support/SupportPackages.tsx

```diff
@@ -0,0 +1,180 @@
+  const { t_i18n, nsdt } = useFormatter();
+  const classes = useStyles();
+  const [commitSupportPackageAdd] = useMutation(supportPackageAddMutation);
+  const { viewStorage, helpers, paginationOptions } = usePaginationLocalStorage<SupportPackageLinesPaginationQuery$variables>(
+    LOCAL_STORAGE_KEY,
+    {
+      searchTerm: '',
+      sortBy: 'name',
+      orderAsc: true,
+    },
+  );
+
+  const generateSupportPackage = () => {
+    const supportPackageName = `support-package-${nsdt(new Date())}`;
+    commitSupportPackageAdd({
+      variables: {
+        input: {
+          name: supportPackageName,
+        },
+      },
+      updater: (store: RecordSourceSelectorProxy) => {
+        insertNode(
+          store,
+          'Pagination_supportPackages',
+          paginationOptions,
+          'supportPackageAdd',
+        );
+      },
+      onCompleted: () => {
+        MESSAGING$.notifySuccess(
+          `Support package request send for ${supportPackageName}.`,
+        );
+      },
+      onError: (error: Error) => {
+        handleError(error);
+      },
+    });
+  };
+
+  const renderLines = () => {
+    const { sortBy, orderAsc, searchTerm } = viewStorage;
+    const queryRef = useQueryLoading<SupportPackageLinesPaginationQuery>(
+      supportPackageLinesQuery,
+      paginationOptions,
+    );
+    const dataColumns = {
+      name: {
+        label: 'Name',
+        width: '39%',
+        isSortable: true,
+      },
+      package_status: {
+        label: 'Status',
+        width: '19%',
+        isSortable: true,
+      },
+      created: {
+        label: 'Date',
+        width: '20%',
+        isSortable: true,
+      },
+    };
+    return (
+      <ListLines
+        helpers={helpers}
+        dataColumns={dataColumns}
+        sortBy={sortBy}
+        orderAsc={orderAsc}
+        keyword={searchTerm}
+        handleSort={helpers.handleSort}
+      >
+        {queryRef && (
+          <React.Suspense>
+            <SupportPackageLines
+              queryRef={queryRef}
+              paginationOptions={paginationOptions}
+              dataColumns={dataColumns}
+              setNumberOfElements={helpers.handleSetNumberOfElements}
+            />
+          </React.Suspense>
+        )}
+      </ListLines>
+    );
+  };
+
+  return (
+    <Security needs={[SETTINGS]} placeholder={<>{t_i18n('You do not have any access to the knowledge of this OpenCTI instance.')}</>}>
+      <div className={classes.container}>
+        <Breadcrumbs variant="list"
+          elements={[{ label: t_i18n('Settings') }, { label: t_i18n('Support packages'), current: true }]}
+        />
+
+        <Grid container={true} spacing={4}
+          classes={{ container: classes.gridContainer }}
+        >
+          <Grid item={true} xs={8} style={{ paddingTop: 10 }}>
+            <div style={{ height: '100%' }}>
+              <div className="clearfix"/>
+              <Paper classes={{ root: classes.paper }} variant="outlined">
```

Comment:
Why do we use a `Paper` component to encapsulate the table here instead of making the same UI from other pages like `Settings/Security`, `Settings/Customization`, `Settings/Taxonomies` and `Settings/Activity`.